ubuntudesign/vanilla-framework
# Enable combining u-text-align--center with u-vertically-center

## Done
- Added the flex alignment styles to the text-align utilities so they work in combination with flex and without it.

## QA
- Pull down this code
- Build with `./run build`
- Create demo.html with the following markup

```html
<html>
<head>
  <title></title>
  <link rel="stylesheet" type="text/css" media="screen" href="build/css/build.css">
</head>
<body>
  <section class="p-strip--dark">
    <div class="row">
        <div class="u-equal-height">
            <div class="col-7">
                <h2>Buy a BootStack managed cloud</h2>
                <p>BootStack gives you a private OpenStack cloud managed by Canonical.  It runs on your hardware, in the datacenter of your choice with our experts responsible for design, deployment and 24x7 support.</p>
                <ul class="p-list">
                    <li class="p-list__item is-ticked">Built to your specifications</li>
                    <li class="p-list__item is-ticked">Your choice of certified hardware</li>
                    <li class="p-list__item is-ticked">Retain complete control of every tool and every machine</li>
                    <li class="p-list__item is-ticked">Guaranteed availability backed by a clear SLA</li>
                    <li class="p-list__item is-ticked">Transfer operations anytime you want</li>
                </ul>
                <p><a href="/cloud/openstack/managed-cloud" class="p-button--positive">Learn more about BootStack</a></p>
            </div>
            <div class="col-5 u-text-align--center u-vertically-center">
              <img src="https://assets.ubuntu.com/v1/466046c7-illustration-cloud-bootstack.svg?w=320" alt="" class="u-hidden--small" width="320">
            </div>
        </div>
    </div>
</section>
</body>
</html>
```

- Check that the image is horizontally and vertically centred.

## Details

I didn't think this needed to be added to documentation as there is now mixed documentation.
